src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,62 @@
+const { expensesService } = require('../services/expenses.service');
+const { usersService } = require('../services/users.service');
+
+class ExpensesController {
+  getAllExpenses = (req, res) => {
+    res.send(expensesService.getAll(req.query));
+  };
+  createExpense = (req, res) => {
+    const { userId } = req.body;
```

Comment:
Consider creating a middleware folder and placing middleware here for checking and validating the user input from the request before passing it to the controller. The user data from the request should be sanitized. Also, you will be able to send 400 errors with additional information about the.